.github/workflows/pkgci_build_packages.yml

```diff
@@ -30,18 +30,19 @@ jobs:
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
         with:
           submodules: true
-      - name: Write version info
-        shell: bash
-        run: |
-          cat << EOF > version_info.json
-          {
-            "package-suffix": "${PACKAGE_SUFFIX}",
-            "package-version": "${{ inputs.package_version }}",
-            "iree-revision": "$(cd ../iree && git rev-parse HEAD)"
-          }
-          EOF
-          realpath version_info.json
-          cat version_info.json
+      # TODO: Generate `version_info_rc.json` instead and include PR number.
```

Comment:
`rc` or `dev`?

Reply:
At the moment the `setup.py` files look for an `version_info_rc.json` first and fallback to a `version_info.json`. The naming `version_info_rc.json` however might not be the best here but that's more some mechanics which we can change easiliy.